package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "leaderboard-project",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "babel-eslint": "^10.1.0",
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.27.5",
+    "hint": "^7.1.10",
+    "html-webpack-plugin": "^5.5.3",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0"
+  }
+}
```

Comment:
- [ ] Kindly make sure all webpack related packages are included in `package.json` like `webpack`, `webpack-cli`, `webpack-dev-server`, And also has the `npm run` commands like `start`, `build` etc... to follow the project requirement.